### ISSUE TITLE
Indices correlation between input and output

### ISSUE THREAD

**Issue by shrit** [LTLA/umappp#17]
Hello,

I am wondering if the indices of the input matrix correspond to the indices of the output matrices, is this the case? or the is some shuffling at the output matrix ?

Best regards,
Omar

**Comment by LTLA** [comment]
Yes, columns of `input` should correspond to columns of `embedding` (i.e., output). No shuffling is done.
